daos_server.yml

```diff
@@ -0,0 +1,36 @@
+access_points: ['replace_this_server:10001']
```

Comment:
Any specific reason why this generic file is being added rather than keeping separate  psm2 and verbs server yaml files?
I personally preferred provider specific files already in the repo rather than having a generic one and going ahead and editing them all the time as per need. But thats my take.

Reply:
I don't think we have a path to support psm2 moving forward with the issues we find in libfabric.
For perf testing on Frontera, we'll be running with verbs.

Reply:
Sounds fine.